libs/shared/contentful/src/lib/queries/eligibility-content-by-offering/query.ts

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { graphql } from '../../../__generated__/gql';
+
+export const eligibilityContentByOfferingQuery = graphql(`
+  query EligibilityContentByOffering(
+    $skip: Int!
+    $limit: Int!
+    $locale: String!
```

Comment:
None of these params are necessary. Only 1 offering should be returned, so no pagination is required, and none of the contents types are localized.